docs/fundamentals/flow/flow-api.md

```diff
@@ -363,5 +240,4 @@ with Flow() as f:
     f.to_k8s_yaml('flow_k8s_configuration')
 ```
 This will generate the necessary Kubernetes configuration files for all the `Executors` of the `Flow`. The generated folder can be used directly with `kubectl` to deploy the `Flow` to an existing Kubernetes cluster.
-More in depth information can be found in {ref}`this How-TO <docker-compos>`.
-
+More in depth information can be found in {ref}`this How-TO <docker-compos>`.
```

Comment:
this HOW-TO should be better linked and with another name perhaps?

Reply:
yes, thinking about a tweak